src/reconcile/rec_col.c

```diff
@@ -1348,6 +1354,10 @@ __wt_rec_col_var(
                 }
                 twp = &vpack->tw;
 
+                /* Clear the on-disk cell time window if it is obsolete. */
+                if (twp->start_ts != WT_TS_NONE)
+                    __wt_rec_time_window_clear_obsolete(session, twp, r);
```

Comment:
Is there a reason to check twp->start_ts here and not for FLCS? I would expect that this will sometimes cause it to not clear the transaction ID when it otherwise could. (Same in rec_row.c, I guess, too.)

Reply:
Yes, it is my mistake. The same check is required for the FLCS is also.

Reply:
Fixed.